app/src/main/java/io/intrepid/contest/screens/contestcreation/categorieslist/CategoriesListFragment.java

```diff
@@ -98,4 +98,16 @@ public Category getDefaultCategory(int categoryNameRes, int categoryDescriptionR
         String categoryDescription = getString(categoryDescriptionRes);
         return new Category(categoryName, categoryDescription);
     }
+
+    @Override
+    public void setNextEnabled(boolean enabled) {
+        ((EditContestContract) getActivity()).setNextEnabled(enabled);
+    }
+
+    @Override
+    public void onFocus() {
+        if (presenter != null) {
```

Comment:
It should still occur whenever a ValidatableView fragment is selected. but only if the presenter has been instantiated. It is not entirely clear to me why the presenter is at times null, but best inference here would be that this is a lifecycle issue. When we get the fragment at some index of our tabAdapter, we can't ascertain anything about which lifecycle events have been fired up for that particular fragment, so we also can't ascertain anything about the presenter for that fragment.  I'm thinking that another way this could work would be to do a check for if the fragment is visible or isAdded then call presenter.onViewBound(); not sure though.